CHANGES.md

```diff
@@ -13,6 +13,8 @@ To be released.
 ### Backward-incompatible storage format changes
 
 ### Added APIs
+- Added `BindingProxies` method. [[756], [#868]]
+- Added `BindingMultipleProxies` method. [#868]
```

Comment:
This statement can be omitted because `NetMQTransport.BindingMultipleProxies()` is private method.